src/useCalendar.ts

```diff
@@ -28,6 +30,7 @@ export function useCalendar({
   months = 1,
   blockFuture,
   allowOutsideDays,
+  locale,
   weekStartsOn = 0,
```

Comment:
we can remove the default value here, this will override the `locale.options.weekStartsOn`